_pytest/capture.py

```diff
@@ -449,8 +449,7 @@ def read(self, *args):
     __iter__ = read
 
     def fileno(self):
-        raise UnsupportedOperation("redirected stdin is pseudofile, "
-                                   "has no fileno()")
+        raise UnsupportedOperation("redirected Stdin is pseudofile, has no fileno()")
```

Comment:
3444796f3e4ba147ccef4df735a8a09ea62b9e48 fixed the upper-case stdin and formatting - looks that got lost somewhere?